src/NotifyIconWpf/TaskbarIcon.cs

```diff
@@ -132,6 +132,7 @@ public TaskbarIcon()
 
             // register event listeners
             messageSink.MouseEventReceived += OnMouseEvent;
+            messageSink.ContextMenuReceived += OnContextMenuEvent;
```

Comment:
I'm not sure if the added redirect via the OnContextMenuEvent adds anything to clear up the code, maybe we just do the following?
`messageSink.ContextMenuReceived += ShowContextMenu;`

Reply:
I don't know whether DPI has any effect on the wParam-Position, so I just added another method here.